June 17 Poll Drafts (#2669)
makerdao/community: governance/polls/SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024.md

---
title: SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024

> summary: Onboard USDC.e (Bridged Circle USDC) to Spark Protocol on Gnosis with included parameters.

Reviewer: ```suggestion
summary: Signal your support or opposition to onboarding USDC.e (Bridged Circle USDC) to Spark Protocol on Gnosis with included parameters.
```

version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2024-06-17T16:00:00
end_date: 2024-06-20T16:00:00
---
# Poll: SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024

The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, June 17 at 16:00 UTC.

**This is a binary vote.**
- **You may vote for a single option.**
- **You should vote for the option which you prefer.**
- **If you would accept either option, you should vote 'Abstain'.**

## Review

This poll allows the MakerDAO governance community to signal their support or opposition to adding USDC.e (Bridged Circle USDC) to Spark Protocol on Gnosis with the parameters below:

### Spark Parameters

* Token: USDC.e
* Token Address: [0x2a22f9c3b484c3629090FeED35F17Ff8F88f76F0](https://gnosisscan.io/token/0x2a22f9c3b484c3629090feed35f17ff8f88f76f0)
* Oracle: 1:1 USD Fixed
* Oracle Provider: N/A
* Borrow Enabled: Yes

> * Flash Loan Enabled	Yes

Reviewer: ```suggestion
* Flash Loan Enabled: Yes
```

* Stable Rate Borrowing: No
* Collateral Enabled: No
* [Isolation Mode](https://docs.spark.fi/defi-infrastructure/sparklend/spark-lend-features#isolation-mode) Enabled: No
* Isolation Mode Debt Ceiling: n/a
* Loan To Value: 0%
* Liquidation Threshold: 0%
* Liquidation Bonus: 0%

> * Liquidation Protocol Fee:	0%

Reviewer: ```suggestion
* Liquidation Protocol Fee: 0%
```

* Base borrow rate: 0%
* Slope1: 9%

> * Slope2:	15%

Reviewer: ```suggestion
* Slope2: 15%
```

> * Token: USDC.e
> * Token Address: [0x2a22f9c3b484c3629090FeED35F17Ff8F88f76F0](https://gnosisscan.io/token/0x2a22f9c3b484c3629090feed35f17ff8f88f76f0)
> * Oracle: 1:1 USD Fixed
> * Oracle Provider: N/A
> * Borrow Enabled: Yes
> * Flash Loan Enabled: Yes
> * Stable Rate Borrowing: No
> * Collateral Enabled: No
> * [Isolation Mode](https://docs.spark.fi/defi-infrastructure/sparklend/spark-lend-features#isolation-mode) Enabled: No
> * Isolation Mode Debt Ceiling: n/a
> * Loan To Value: 0%
> * Liquidation Threshold: 0%
> * Liquidation Bonus: 0%
> * Liquidation Protocol Fee: 0%
> * Base borrow rate: 0%
> * Slope1: 9%
> * Slope2: 15%
> * Optimal Utilization: 95%
> * Reserve factor: 10%
> * Borrowable in isolation mode: Yes
> * [Siloed borrowing](https://docs.spark.fi/defi-infrastructure/sparklend#siloed-borrowing): No
> * [Efficency Mode](https://docs.spark.fi/defi-infrastructure/sparklend#efficiency-mode-emode) "Emode" Category:	0
> * [Isolation Mode](https://docs.sparkprotocol.io/developers/features/isolation-mode) Enabled: N/A
> * Isolation Mode Debt Ceiling: N/A
> * [Supply cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps): 10,000,000 USDC.e
> * [Borrow cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps):  8,000,000 USDC.e

Reviewer: ```suggestion
* Token: **USDC.e**
* Token Address: **[0x2a22f9c3b484c3629090FeED35F17Ff8F88f76F0](https://gnosisscan.io/token/0x2a22f9c3b484c3629090feed35f17ff8f88f76f0)**
* Oracle: 1:1 USD Fixed
* Oracle Provider: N/A
* Borrow Enabled: **Yes**
* Flash Loan Enabled: **Yes**
* Stable Rate Borrowing: **No**
* Collateral Enabled: **No**
* [Isolation Mode](https://docs.spark.fi/defi-infrastructure/sparklend/spark-lend-features#isolation-mode) Enabled: **No**
* Isolation Mode Debt Ceiling: n/a
* Loan To Value: **0%**
* Liquidation Threshold: **0%**
* Liquidation Bonus: **0%**
* Liquidation Protocol Fee: **0%**
* Base borrow rate: **0%**
* Slope1: **9%**
* Slope2: **15%**
* Optimal Utilization: **95%**
* Reserve factor: **10%**
* Borrowable in isolation mode: **Yes**
* [Siloed borrowing](https://docs.spark.fi/defi-infrastructure/sparklend#siloed-borrowing): **No**
* [Efficency Mode](https://docs.spark.fi/defi-infrastructure/sparklend#efficiency-mode-emode) "Emode" Category:	**0**
* [Isolation Mode](https://docs.sparkprotocol.io/developers/features/isolation-mode) Enabled: N/A
* Isolation Mode Debt Ceiling: N/A
* [Supply cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps): **10,000,000 USDC.e**
* [Borrow cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps):  **8,000,000 USDC.e**
```
